layouts/shortcodes/feature-gate-list.html

```diff
@@ -0,0 +1,130 @@
+{{/* Kubernetes major version is hard-coded as 1 */}}
```

Comment:
The major version is 1 today. It is not hard-coded as anything afaict.

Reply:
I can clarify to explain that this code assumes Kubernetes' major version has never been above 1.